app/src/main/java/com/example/fundatecheroes/character/response/CharacterResponse.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.fundatecheroes.character.response
+
+data class CharacterResponse(
+    val id: Int,
+    val name: String,
+    val description: String,
+    val image: String,
+    val universeType: String,
+    val characterType: String,
+    val age: Int,
+    val birthday: String?
+)
+{
+    enum class MarvelDC { MARVEL, DC }
+    enum class HeroVillain { HERO, VILLAIN}
```

Comment:
Apenas uma sugestão de nomenclatura
```suggestion
    enum class COMIC_TYPE { MARVEL, DC }
    enum class CHARACTER_TYPE { HERO, VILLAIN}
```